client/src/assets/locales/en.ts

```diff
@@ -41,6 +41,8 @@ export const en = {
       en: "Switch to English",
       es: "Cambiar a Español",
     },
+    close_menu_aria_label: "Close menu",
+    open_menu_aria_label: "Open menu",
```

Comment:
Do these also need to appear somewhere else?

Reply:
Sorry for vagueness, I meant does these texts need to be translated and/or be added to some other repo?